api/v1alpha1/kafkauser_types.go

```diff
@@ -96,3 +103,10 @@ func (spec *KafkaUserSpec) GetIfCertShouldBeCreated() bool {
 func (spec *KafkaUserSpec) GetAnnotations() map[string]string {
 	return util.CloneMap(spec.Annotations)
 }
+
+func (spec *KafkaUserSpec) GetExpirationSeconds() int32 {
+	if spec.ExpirationSeconds == nil {
+		return int32(certmanagerv1.DefaultCertificateDuration.Seconds())
```

Comment:
I would use a const value here 90 days. 
We will not know about if cert manager change the default value to something else and our description become false

Reply:
What do you think @panyuenlau ?

Reply:
@bartam1 I actually proposed to use what @shubhamcoc has here, the reason was to more clearly indicate where this 90 days comes from - it's from cert-manager rather than Koperator.

If cert-manager changes this default value in the future, we don't need to do anything (assuming we have the dependency updated) as long as cert-manager itself still uses the same const in their library - this was actually another reason that I proposed the change

Reply:
I understand but who will change the value in the CRD description (When it is not specified the default validation duration is 90 days) I think it will never change anyway so I let it go.